config/services.yml

```diff
@@ -499,15 +529,7 @@ https://cmr.uat.earthdata.nasa.gov:
           STAGING_PATH: public/sds/swot-reproject
     umm_s:
       - S1237974711-EEDTEST
-    collections:
-      # DO NOT UNCOMMENT THE FOLLOWING LINE - this collection is added via UMM-S and is left here
-      # to prevent re-adding. This is needed for tests related to HARMONY-875
-      # - C1233860183-EEDTEST
-      - C1233800302-EEDTEST
-      - C1221286860-NSIDC_TS1
-      - C1226288788-EDF_DEV07
-      - C1233147317-EEDTEST
-      - C1233147319-EEDTEST
+    collections: []
```

Comment:
Note to us (you don't need to change this PR) - I'm thinking we may as well change the code to make collections optional and treat it as empty if missing.

Reply:
Yeah - my first guess was to remove `collections` altogether, but that didn't work so well (hence the second commit). That would be a nice change. It definitely felt nice to clean out some lines from this file, though!